src/scene/constants.js

```diff
@@ -617,9 +617,10 @@ export const SHADOWUPDATE_REALTIME = 2;
 export const SORTKEY_FORWARD = 0;
 export const SORTKEY_DEPTH = 1;
 
-export const MASK_DYNAMIC = 1;
-export const MASK_BAKED = 2;
-export const MASK_LIGHTMAP = 4;
+// flags used on the mask property of the Light, and also on mask property of the MeshInstance
+export const MASK_AFFECT_DYNAMIC = 1;
+export const MASK_AFFECT_LIGHTMAPPED = 2;
```

Comment:
Is the engine stadnard `ENUM_TYPE_ENUM_VALUE` or `ENUMTYPE_ENUMVALUE`? While I can find some of the former, the latter seems way more prevalent.

Reply:
personally I prefer the more readable version with the extra _

Reply:
OK, so are you advocating migrating the engine's enum constants to the more liberal use of underscore?

Reply:
I guess so .. looking at it again, it seems more than 50% already use underscores for longer constants .. but I'm not sure it's worth adjusting the rest, especially public constants?

Reply:
:shrug: Just pointing out this is definitely an area of inconsistency in the engine right now.

Reply:
TBH, I wish we just used literal strings for most constants.